doc/Developing.md

```diff
@@ -513,6 +513,12 @@ etc.) by opening `packages/<package name>/coverage/index.html`.
  - Install Node.js with `nvm`: `nvm install 12.14.1`, then use it: `nvm use 12.14.1`. You can list all available Node.js versions with `nvm list available` if you want to pick another version.
  - Install `yarn`: `scoop install yarn`.
  - If you need to install `windows-build-tools`, see [`Installing Windows Build Tools`](#installing-windows-build-tools).
+ - If you run into problems with installing the required build tools, the `node-gyp` documentation offers a useful [guide](https://github.com/nodejs/node-gyp#on-windows) how to install the dependencies manually. The versions required for building Theia are:
+   - The python version should be 3.6 or higher
+   - Visual Studio 2017 for installing the Visual C++ Build Environment
```

Comment:
I don't remember: do I have to get the Visual Studio installer to install the build environment? Also: I read something that gyp can now install the deps itself. True or false?

Reply:
In would think the artifact needed is this one: https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022.

Reply:
```suggestion
- Visual Studio [build tools](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022) 17
```